### PR TITLE
Update nodejs post install message

### PR DESCRIPTION
**Pull request by tschf**
As discussed in #335 

> Sounds good to me! I would accept a good patch changing this terminology + adapting tests ;)

Not sure what to do for adapting tests. Can you offer any guidance here?
I tried running tests (`./runtests small`, `./runtests medium`) even before any changes (both with steps in README for virtualenv or just installing dependencies on core system) and didn't have success with those. This probably is the right place to discuss those issues, but can provide more info.

Output after the change:

```bash
trent@birroth:~/Projects/ubuntu-make$ ./bin/umake nodejs
Download from https://nodejs.org/dist/latest/SHASUMS256.txt
Choose installation path: /home/trent/.local/share/umake/nodejs/nodejs-lang
Downloading and installing requirements                                                                                                                                                                            |
100% |#############################################################################################################################################################################################################|
Installing Nodejs Lang
|##################################################################################################################################################################################################################|
You may need to log back in for your Nodejs Lang installation to work properly
Installation done
```

cc @didrocks

### CODE REVIEW

**Comment by didrocks** [comment]
Hey! Thanks for the patch, the changes looks good to me!

After checking anyway, there is no tests that ensure that this is written down (which is ok, as just an user information message).
However, do you mind applying the same changes to other frameworks? scala, go, android, switch, kotlkin, dart & rust are using those. Would be better to have all of them using the same message :)

Thanks in advance!

**Comment by tschf** [comment]
I thought it might not be a bad idea to declare the message once (baseinstaller.py) and reference that from the sub-classes. Or do you prefer to keep them defined in each relevant framework?

Manually tested on:

- android
- dart
- go
- kotlin
- nodejs
- rust
- scala
- swift

File: umake/frameworks/android.py [hunk 1]
```diff
@@ -136,8 +136,7 @@ def post_install(self):
         # the SDK manager
         add_env_to_user(self.name, {"PATH": {"value": [os.path.join("$ANDROID_HOME", "tools"),
                                                        os.path.join("$ANDROID_HOME", "platform-tools")]}})
-        UI.delayed_display(DisplayMessage(_("You need to restart your current shell session for your {} installation "
-                                            "to work properly").format(self.name)))
+        UI.delayed_display(DisplayMessage(_(super().POST_INSTALL_WARN).format(self.name)))
```
Comment (didrocks) [inline umake/frameworks/android.py]: as it's inherited, you can replace `super()` by `self`, isn't it?

File: umake/frameworks/baseinstaller.py [hunk 2]
```diff
@@ -41,6 +41,11 @@
 class BaseInstaller(umake.frameworks.BaseFramework):
 
     DIRECT_COPY_EXT = ['.svg', '.png', '.ico', '.jpg', '.jpeg']
+    # Framework environment variables are added to `~/.profile` which may
+    # require logging back into your session for the changes to be picked up.
+    # Use `POST_INSTALL_WARN` to alert users to this fact, in `post_install`
+    # function. {} in replaced with the framework name at runtime.
+    POST_INSTALL_WARN = "You may need to log back in for your {} installation to work properly"
```
Comment (didrocks) [inline umake/frameworks/baseinstaller.py]: I would change the variable name to `RELOGIN_REQUIRE_MSG`.

Also, this is the variable that needs to have = _("…"). So, you can remove it from the other calls, and only have it once for translation here (the other calls isn't useful, as gettext won't be able to pick up the text assignement properly).

File: umake/frameworks/android.py (hunk 1 above)
Comment (tschf) [inline umake/frameworks/android.py]: Ah yeah - I wasn't sure, and had tested with `super()` first so went with that. Will update.

**Comment by tschf** [comment]
Just pushed those requested changes.

**Comment by didrocks** [comment]
Reviewed the changes and everything looks good (and small tests pass!). Good work, thanks again for your contribution there! :+1: 
Merging :)

**Comment by tschf** [comment]
No probs! Thanks for the great tool!

**Pull request merged**